index.html

```diff
@@ -0,0 +1,328 @@
+            <div class="grid-item item2">
+                <div class="content">
+                    <h1 class="p-text">Professional Art <br>
+                        Printing Data
+                    </h1>
+                    <h2 class="s-text">
+                        A daily selection of privately
+                        personalized reads; no accounts or
+                        sign-ups required. Has been the 
+                        industry's standard.
+                    </h2>
+                    <div class="tags">
+                        <ul class="tag">
+                            <li> <a href="#">HTML</a></li>
+                            <li> <a href="#">Bootstrap</a></li>
+                            <li> <a href="#">Ruby</a></li>
+                        </ul>
+                    </div>
+
+                    <div class="btn2-holder">
+                        <a href="#" class="btn2">See Projects</a>
+                    </div>
+                </div>
+            </div>
+            <div class="grid-item item2">
+                <div class="content">
+                    <h1 class="p-text">Professional Art <br>
+                        Printing Data
+                    </h1>
+                    <h2 class="s-text">
+                        A daily selection of privately
+                        personalized reads; no accounts or
+                        sign-ups required. Has been the 
+                        industry's standard.
+                    </h2>
+                    <div class="tags">
+                        <ul class="tag">
+                            <li> <a href="#">HTML</a></li>
+                            <li> <a href="#">Bootstrap</a></li>
+                            <li> <a href="#">Ruby</a></li>
+                        </ul>
+                    </div>
+
+                    <div class="btn2-holder">
+                        <a href="#" class="btn2">See Projects</a>
+                    </div>
+                </div>
+            </div>
+            <div class="grid-item item2">
+                <div class="content">
+                    <h1 class="p-text">Professional Art <br>
+                        Printing Data
+                    </h1>
+                    <h2 class="s-text">
+                        A daily selection of privately
+                        personalized reads; no accounts or
+                        sign-ups required. Has been the 
+                        industry's standard.
+                    </h2>
+                    <div class="tags">
+                        <ul class="tag">
+                            <li> <a href="#">HTML</a></li>
+                            <li> <a href="#">Bootstrap</a></li>
+                            <li> <a href="#">Ruby</a></li>
+                        </ul>
+                    </div>
+
+                    <div class="btn2-holder">
+                        <a href="#" class="btn2">See Projects</a>
+                    </div>
```

Comment:
> - [ ] Please kindly add the hover and pressed button state interactions to all your "See project" buttons to match the Figma design. This will give a better user experience for your website👍
> 
> Figma design Hover and Pressed state:
> 
> ![see1](https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/40009354/ac4589a5-c774-4e37-89ee-94a1cbd44711)
> 
> Your design on Hover and Pressed:
> 
> ![see](https://github.com/Bettenoch/Portfolio-Setup-Mobile-First/assets/40009354/373babcf-6558-42d2-8c20-5e676faf505a)
> 
>